docs/resources/servergroup.md

```diff
@@ -12,29 +11,139 @@ Represent server group resource
 
 ## Example Usage
 
+
+### Prerequisite
+
 ```terraform
 provider gcore {
   permanent_api_token = "251$d3361.............1b35f26d8"
 }
 
-resource "gcore_servergroup" "default" {
+data "gcore_project" "project" {
+  name = "Default"
+}
+
+data "gcore_region" "region" {
+  name = "Luxembourg-2"
+}
+```
+
+### Creating basic server group
+
+```terraform
+resource "gcore_servergroup" "servergroup" {
+  name       = "my-servergroup"
+  policy     = "affinity"
+  project_id = data.gcore_project.project.id
+  region_id  = data.gcore_region.region.id
+}
+```
+
+### Creating server group with two instances
+
+```terraform
+data "gcore_image" "ubuntu" {
+  name       = "ubuntu-22.04-x64"
+  region_id  = data.gcore_region.region.id
+  project_id = data.gcore_project.project.id
+}
+
+resource "gcore_volume" "boot_volume" {
+  name       = "my-boot-volume"
+  type_name  = "ssd_hiiops"
+  size       = 5
+  image_id   = data.gcore_image.ubuntu.id
+  project_id = data.gcore_project.project.id
+  region_id  = data.gcore_region.region.id
+}
+
+resource "gcore_volume" "boot_volume2" {
+  name       = "my-boot-volume2"
+  type_name  = "ssd_hiiops"
+  size       = 5
+  image_id   = data.gcore_image.ubuntu.id
+  project_id = data.gcore_project.project.id
+  region_id  = data.gcore_region.region.id
+}
+
+resource "gcore_instancev2" "instance" {
+  flavor_id     = "g1-standard-2-4"
+  name          = "my-instance"
+
+  volume {
+    volume_id  = gcore_volume.boot_volume.id
+    boot_index = 0
+  }
+
+  interface {
+    type = "external"
+    name = "my-external-interface"
+  }
+
+  project_id = data.gcore_project.project.id
+  region_id  = data.gcore_region.region.id
+}
+
+resource "gcore_instancev2" "instance2" {
+  flavor_id     = "g1-standard-2-4"
+  name          = "my-instance2"
+
+  volume {
+    volume_id  = gcore_volume.boot_volume.id
+    boot_index = 0
+  }
+
+  interface {
+    type = "external"
+    name = "my-external-interface"
```

Comment:
should it be "my-external-interface2"?

Reply:
or should it be unique only inside one instance?

Reply:
It should be unique only inside one instance